### ISSUE TITLE
[Feature Request] Include pictures from the PDF

### ISSUE THREAD

**Issue by dummifiedme** [akaalias/obsidian-extract-pdf#7]
Its a kind request for this plugin to have a way to extract images as well. It could be made optional for a user, but a quick extraction with images and graphs would really help :)

**Comment by mochsner** [comment]
Yeah I would appreciate this as well. I think as part of this, since images are what make PDFs so large in size, it might also be helpful to log the image names and their associated file size -- I imagine that without that, crashes might seem like a plugin issue.

**Comment by mochsner** [comment]
Note: this seems like a feature request in the dependency for this project, I believe to be the following project:
https://github.com/opendocsg/pdf2md-web (node)

(that npm project was derived from https://github.com/jzillmann/pdf-to-markdown)

**Comment by dummifiedme** [comment]
Even without extracting images per say, we can just allow kind of an screenshot of the desired area. Image or not. Eg drawing a rectangle over an area could mark it for extraction. 

I discussed it [here](https://forum.obsidian.md/t/discussion-extracting-annotations-from-pdfs/24411) as well.
